src/xenia/apu/xma_decoder.cc

```diff
@@ -55,7 +55,34 @@ XmaDecoder::XmaDecoder(cpu::Processor* processor)
 XmaDecoder::~XmaDecoder() = default;
 
 void av_log_callback(void* avcl, int level, const char* fmt, va_list va) {
-  xe::LogLineVarargs('A', fmt, va);
+#ifdef NDEBUG
```

Comment:
Would be nice to have this as a flag instead (like --verbose_libav_logging or something).